cmd/ci-scheduling-webhook/cloudfront/generate_range_array.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+from typing import List, Dict, Tuple
```

Comment:
Everything in cloudfront/ should be removed. This content is in https://github.com/openshift/ci-tools/pull/3429 .